base/db/tests/testthat/test.check.missing.files.R

```diff
@@ -0,0 +1,19 @@
+test_that("`check_missing_files()` able to return correct missing files", {
+    # Mock `purrr::map_dfr`
+    mocked_res <- mockery::mock(data.frame(file = c("A", "B"), file_size = c(100, 200), missing = c(FALSE, FALSE), empty = c(FALSE, FALSE)))
+    mockery::stub(check_missing_files, "purrr::map_dfr", mocked_res)
```

Comment:
I think what happens inside map_dfr is part of what we want to be testing here -- might be better to stub out `file.size` since that's the only place that would be looking at the filesystem.
```suggestion
    mocked_size <- mockery::mock(100,200)
    mockery::stub(check_missing_files, "file.size", mocked_res)
```

Reply:
It would also be good to include cases with missing and empty files and confirm that it produces the expected error